### PR TITLE
Test Aruba failure

### PR DESCRIPTION
**Pull request by mvz**
This is just an attempt to figure out what's going on ...

### CODE REVIEW

**Comment by JonRowe** [comment]
Thanks for the help debugging this @mvz, any tips on how to find out what its happening underneath are appreciated!

For what its worth my working theory is that perhaps travis is `kill -9`ing the selenium / chrome driver process as we exit and this is bubbling up to us, I can say that the rspec scenario *is* passing, so this status is coming from an interaction of the processes involved... RSpec itself would not return `9` unless configured to do so which it is not in this suite.

**Comment by benoittgt** [comment]
I found a way to fix it. https://github.com/rspec/rspec-rails/commit/b628615073f6514ebf655cc74724de74292f5a72

But I am not able to explain why we had a status 9 when using cucumber and not when running the test directly. I am not sure we need to spend more time on this... even if I am curious about the answer. :)

**Comment by mvz** [comment]
That's great news @benoittgt.

File: features/system_specs/system_specs.feature [hunk 1]
```diff
@@ -82,6 +82,7 @@ Feature: System spec
       When I run `rspec spec/system/some_job_system_spec.rb`
       Then the example should pass
 
+    @announce-output
```
Comment (benoittgt) [inline features/system_specs/system_specs.feature:85]: This is a very good tips. Thanks a lot.

**Pull request closed without merging**